cluster/addons/cluster-monitoring/standalone/heapster-controller.yaml

```diff
@@ -40,7 +40,6 @@ spec:
           command:
             - /heapster
             - --source=kubernetes.summary_api:''
-            - --metric_resolution=60s
```

Comment:
I changed it to be 60s by default in Heapster https://github.com/kubernetes/heapster/pull/1164.
There is not much sense to have it different and overwrite in config.